sonic-net/sonic-buildimage
# [FRR][patch] Fix syntax error for get correctly value

#### Why I did it
There is a syntax error, so the value cannot be retrieved correctly. As a result, a wrong value (null) is written to the specified Python file.

This error causes the specified Python file to not know which Python version to use for execution.
##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
Fix the syntax error for get correctly value
#### How to verify it
1. build frr_9.0.1-sonic-0_amd64.deb
2. Check below 4 files exist '#!/<path of python3>' in the head line
   a. debian/tmp/usr/lib/frr/frr-reload.py
   b. debian/tmp/usr/lib/frr/generate_support_bundle.py
   c. debian/tmp/usr/lib/frr/frr_babeltrace.py
   d. debian/tmp/usr/lib/frr/ospfclient.py


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
